docs/synapse/userguides/storm_adv_control.rstorm

```diff
@@ -198,37 +337,51 @@ If-Elif-Else
   elif <condition> { <storm> }
   else { <storm> }
 
-If ``<condition>`` is met, execute the associated Storm; otherwise, if (else if) the second ``<condition>`` is met, execute the associated Storm; otherwise (else) execute the final Storm query.
+If ``<condition>`` is met, execute the associated Storm; otherwise, if (else if) the second ``<condition>`` is met,
+execute the associated Storm; otherwise (else) execute the final Storm query.
 
-You can use multiple ``elif`` statements before the final ``else``. ``If-elif-else`` is helpful because it allows you to handle multiple conditions differently while avoiding "nested" ``if-else`` statements.
+You can use multiple ``elif`` statements before the final ``else``. ``If-elif-else`` is helpful because it allows
+you to handle multiple conditions differently while avoiding "nested" ``if-else`` statements.
 
 
 **Example:**
 
-You have a subscription to a third-party malware service that allows you to download malware binaries via the service's API. However, the service has a query limit, so you don't want to make any unnecessary API requests that might exhaust your limit.
+You have a subscription to a third-party malware service that allows you to download malware binaries via the
+service's API. However, the service has a query limit, so you don't want to make any unnecessary API requests that
+might exhaust your limit.
 
-You can use a simple if-else statement to check whether you already have a copy of the binary in your storage Axon before attempting to download it.
+You can use a simple if-else statement to check whether you already have a copy of the binary in your storage Axon
+before attempting to download it.
+
+::
+  
+  <inbound file:bytes node(s)>
+  
+  if $lib.bytes.has(:sha256) { }
+  
+  else { | malware.download }
```

Comment:
the pipe before `malware.download` isn't actually needed here

Reply:
It's an "analyst consistency habit" - easier to explain to people "use a pipe when switching between Storm operations and commands, and between individual commands" then to caveat that you don't technically need the pipe for the first command. Since these docs are "user facing" (and other examples use this convention) I lean towards keeping, but could go either way. Thoughts?

Reply:
gotcha, yeah that's fine, it doesn't hurt anything being there